tests/e2e/pageObjects/add-redis-database-page.ts

```diff
@@ -38,7 +39,7 @@ export class AddRedisDatabasePage {
       //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
       //-------------------------------------------------------------------------------------------
       //BUTTONS
-      this.addDatabaseButton = Selector('[data-testid=add-redis-database]');
+      this.addDatabaseButton = Selector('[data-testid=add-redis-database]', {timeout: 1000});
```

Comment:
We are not adding it here generally, just within the common action functions/tests

Reply:
removed